src/libs/actions/PersonalDetails.js

```diff
@@ -183,6 +184,14 @@ function getFromReportParticipants(reports) {
 // When the app reconnects from being offline, fetch all of the personal details
 NetworkConnection.onReconnect(fetch);
 
+// Refresh the personal details every 30 minutes because there is no
+// pusher event that sends updated personal details data yet
+// See https://github.com/Expensify/ReactNativeChat/issues/468
+setInterval(fetch, 1000 * 60 * 30);
+
+// Refresh the timezone every 30 minutes
+setInterval(fetchTimezone, 1000 * 60 * 30);
```

Comment:
NAB we could also just have one timer that handles both of these calls.

```js
setInterval(() => {
    if (isOffline) {
        return;
    }

    fetch();
    fetchTimezone();
}, 1000 * 60 * 30);

Reply:
Oh I like this, it's a better reason to move the isOffline check to the interval too 😄